system/jlib/jlog.hpp

```diff
@@ -153,6 +160,20 @@ inline const char * LogMsgAudienceToFixString(LogMsgAudience audience)
         return("UNKNOWN  ");
     }
 }
+inline unsigned LogMsgAudFromAbbrev(char const * abbrev)
```

Comment:
trivial: our normal casing for method would be camelCase with lower case start.
i.e. LogMsgFieldsFromAbbrevs() should really have been logMsgFieldsFromAbbrevs()

Would be a good time as you're adding new ones to make them and it cased normally.